RELEASING.md

```diff
@@ -57,8 +51,11 @@ You'll also need to have permissions to push directly to the `master` branch.
        python3 setup.py sdist  # create a source distribution
        python3 setup.py bdist_wheel  # create a universal wheel
 
+1. Attach the resulting binaries in (`dist/snorkel-x.x.x.*`) to the release.
+
 1. Check that everything looks correct by uploading the package to the PyPI test server:
 
+       pip install twine  # if not installed
```

Comment:
update conda instructions below to say PR is auto-submitted and a maintainer needs to accept